packages/react/src/DevGrid/DevGrid.module.css

```diff
@@ -0,0 +1,43 @@
+:root {
+  --scrollbar-width: 0px;
+  --devGrid-width: 90%;
+  --devGrid-max-width: 80rem;
+  --devGrid-grid-size: 12;
+  --devGrid-column-gap: var(--brand-stack-gap-normal);
+  --devGrid-column-gap-mobile: var(--brand-stack-gap-condensed);
+  --devGrid-column-color: rgba(255, 0, 0, 0.1);
+  --devGrid-column-border: rgba(255, 0, 0, 1);
+  --devGrid-zIndex: 999; /* reasonably safe high z-index */
+}
+
+.DevGrid {
+  bottom: 0;
+  column-gap: var(--devGrid-column-gap-mobile);
+  display: grid;
+  grid-template-columns: repeat(var(--devGrid-grid-size), 1fr);
+  height: 100%;
+  left: 50%;
+  max-width: var(--devGrid-max-width);
+  pointer-events: none;
+  position: fixed;
+  right: auto;
+  top: 0;
+  transform: translateX(-200vw);
+  width: var(--devGrid-width);
+  z-index: var(--devGrid-zIndex);
+}
+
+.DevGrid--show {
+  transform: translateX(-50%);
+}
+
+@media screen and (min-width: 768px) {
+  .DevGrid {
+    column-gap: var(--devGrid-column-gap);
```

Comment:
wonder if we can do this through a token instead. what do you think? Check out typography components for an example.